Dockerfile

```diff
@@ -6,5 +6,11 @@ RUN microdnf install -y python38 python38-devel curl python3-pip git tar xz bzip
 COPY src src
 COPY poetry.lock poetry.lock
 COPY pyproject.toml pyproject.toml
+COPY unit_test.sh unit_test.sh
+COPY requirements.txt requirements.txt
+COPY dev dev
+COPY tests tests
+COPY cdappconfig.json cdappconfig.json
+COPY parse_json.py parse_json.py
```

Comment:
Can we shift `cdappconfig.json` and `parse_json.py` under dev directory as these files are not related to the production code. Doing this will reduce Dockerfile lines and image layers while building the image. 
Also please shift line 6 `COPY src src` after `RUN pip3 install` this will reduce the image build time (while building docker will use old unchanged image layers).

Reply:
Thanks for the review! Refactored the code.